src/core/occupancy.ts

```diff
@@ -22,12 +13,16 @@ import EventEmitter, { wrap } from './utils/event-emitter.js';
  *
  * Get an instance via {@link Room.occupancy}.
  */
-export interface Occupancy extends EmitsDiscontinuities {
+export interface Occupancy {
   /**
    * Subscribe a given listener to occupancy updates of the chat room.
    *
+   * Note: This requires occupancy events to be enabled via the `enableEvents` option in
+   * the room options. If this is not enabled, an error will be thrown.
```

Comment:
```suggestion
   * Note: This requires occupancy events to be enabled via the `occupancy.enableEvents` option in
   * the room options. If this is not enabled, an error will be thrown.
```

or ... "in the _occupnacy_ room options"